src/SearchResults.js

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import moment from "moment";
+// import { useState } from "react";
+
+const data = [
+  {
+    title: "",
+    firstName: "",
+    surname: "",
+    Email: "",
+    Room: "",
+    CheckInDate: "",
+    CheckOutDate: "",
+    stayNights: "",
+  },
+];
+function SearchResults({ bookingResults }) {
+  const columnKey = [
+    "id",
+    "title",
+    "firstName",
+    "surname",
+    "email",
+    "roomId",
+    "checkInDate",
+    "checkOutDate",
+    "nightsIn",
+  ];
+  return (
+    <table class="table table-striped table-dark">
+      <thead>
+        <tr className="title-table">
+          <th scope="col">#</th>
+          {columnKey.map((column) => (
+            <th scope="col">{column}</th>
+          ))}
+        </tr>
+      </thead>
+      <tbody>
+        {bookingResults.map((result, index) => (
+          <tr>
+            <th scope="row">{index + 1}</th>
+            <td>{result.id}</td>
+            <td>{result.title}</td>
+            <td>{result.firstName}</td>
+            <td>{result.surname}</td>
+            <td>{result.email}</td>
+            <td>{result.roomId}</td>
+            <td>{result.checkInDate}</td>
+            <td>{result.checkOutDate}</td>
+            <tg>{result.stayNights}</tg>
```

Comment:
What's this tag? I guess it's a typo. You probably are getting an error in dev tool of browser if you run `SearchResults`

Reply:
It was a typo ☺️ Thank you